README.md

```diff
@@ -66,11 +70,20 @@ jobs:
 
 ### Buildpacks - Release
 
-Prepares a buildpack release by:
-* bumping the fixed version
-* updating changelogs
-* generating an aggregate changelog from all the changelogs
-* opening a PR against the repository with the modified files
+Prepares one or more buildpacks release by:
+
+* Detects all the buildpacks in a repository and compiles them into Cloud Native Buildpacks
+* For each compiled buildpack:
+  * Creates a CNB archive file from the compiled buildpack and publishes it as a GitHub Release
+  * Creates an OCI image from the compiled buildpack and publishes it to Docker
```

Comment:
(Feel free to reword this suggestion further - it's still missing an explanation of `[metadata.release]` etc, though maybe that's good enough for now?)

```suggestion
  * Creates an OCI image from the compiled buildpack and publishes it to the Docker Hub repository specified in the buildpack's `buildpack.toml`
```

Reply:
added information about the metadata key used